### DISCUSSION TITLE
Dropdownlist doesn't display the Textfield but the Id is stored to the Database

### DISCUSSION THREAD

**Discussion by imhemayatsangin** [serenity-is/Serenity#6001] [Help] [answered]
I have a select box or we can say a dropdown where it fills the employee name and my code is like this in the ContractRow.cs:
`   [DisplayName("Employee"), Column("EmployeeID"), ForeignKey("[HR].[Employee]", "EmpID"), LeftJoin("jEmployee"),TextualField("EmployeeLastname")]
        [LookupEditor("HR.Employee")]
        public Int32? EmployeeId
        {
            get => fields.EmployeeId[this];
            set => fields.EmployeeId[this] = value;
        }`
and also:
```
  [DisplayName("Employee Lastname"), Expression("jEmployee.[Lastname]")]
        public String EmployeeLastname
        {
            get => fields.EmployeeLastname[this];
            set => fields.EmployeeLastname[this] = value;
        }
```
and 
```
public class RowFields : RowFieldsBase
        {
 public StringField EmployeeLastname;
}
```
When im trying to select the value i can understand that the field has a value but the text is not showing up and when i save the data the id also stores to the database which is an employee id. 
Please if anybody can help me out what to do with.
Note: I'm using some other dropdown boxes in the same form and that is working fine.
Looking forward for your response.
Thanks.

**Comment by arunkumar12712** [comment, marked as answer]
Expression use like this 
Expression("Concat(PartyName,' [M:', ContactNo, ']')")]
